express-nosql/podman-compose.yml

```diff
@@ -8,4 +29,8 @@ services:
       - MONGO_INITDB_ROOT_USERNAME=root
       - MONGO_INITDB_ROOT_PASSWORD=nodegmp
     ports:
-      - '27017:27017'
\ No newline at end of file
+      - '27017:27017'
```

Comment:
you dont need to expose ports for mongodb  becuase it will be only used by app container. containers that are on same network can communicate without exposing port